js/search.js

```diff
@@ -0,0 +1,70 @@
+document.querySelector('.searchBtn').addEventListener('click', (e) => {
+    console.log('searching');
+    const query = document.querySelector('#search-input').value;
+    const result = Store.searchBook(query);
+    UI.clearBooks();
+    UI.displayBooks(result); 
+  });
+
+  document.querySelector('.searchBtn').addEventListener('click', (e) => {
+    console.log('searching');
+    const query = document.querySelector('#search-input').value;
+    const result = Store.searchBook(query);
+    UI.clearBooks();
+    UI.displayBooks(result); 
+  });
```

Comment:
This seems like redundant/duplicate code, similar to the one above it line(1-7).